applications/luci-app-vpnbypass/luasrc/model/cbi/vpnbypass.lua

```diff
@@ -1,18 +1,39 @@
-readmeURL = "https://github.com/openwrt/packages/blob/master/net/vpnbypass/files/README.md"
+local readmeURL = "https://github.com/openwrt/packages/blob/master/net/vpnbypass/files/README.md"
 
 m = Map("vpnbypass", translate("VPN Bypass Settings"))
-s = m:section(NamedSection, "config", "vpnbypass")
-
--- General options
-e = s:option(Flag, "enabled", translate("Start VPNBypass service"))
-e.rmempty = false
-function e.write(self, section, value)
-	if value ~= "1" then
-		luci.sys.init.stop("vpnbypass")
+
+h = m:section(NamedSection, "config", "vpnbypass", translate("Service Status"))
+local packageName = "vpnbypass"
+local uci = require "luci.model.uci".cursor()
+local sys = require "luci.sys"
+local http = require "luci.http"
+local dispatcher = require "luci.dispatcher"
+en = h:option(Button, "__toggle")
+if enabledFlag ~= "1" then
+	en.title      = translate("Service is disabled/stopped")
+	en.inputtitle = translate("Enable/Start")
+	en.inputstyle = "apply important"
+else
+	en.title      = translate("Service is enabled/started")
+	en.inputtitle = translate("Stop/Disable")
+	en.inputstyle = "reset important"
+end
+function en.write()
+	enabledFlag = enabledFlag == "1" and "0" or "1"
+	uci:set(packageName, "config", "enabled", enabledFlag)
+	uci:save(packageName)
+	uci:commit(packageName)
+	if enabledFlag == "0" then
+		sys.init.stop(packageName)
+	else
```

Comment:
Wouldn't we also have to disable at this point?

Reply:
You're right. Since I use scripts, it's easier for me to resubmit PR than fix this one, I hope it's OK.

Reply:
no problem